CHANGELOG.md

```diff
@@ -2,19 +2,35 @@
 
 ## Unreleased
 
+## 3.6.0 (Feature release)
+
 ### New features
 
-#### Add display override for hiding content when printing
+#### Use colours with the tag component
+
+You can now [use colour with tags](https://design-system.service.gov.uk/components/tag/#using-colour-with-tags) to help distinguish between different tags - or to help draw the user’s attention to a tag if it’s especially important.
+
+This also means you should replace the `.govuk-tag--inactive` class with the `.govuk-tag--grey` class. `.govuk-tag--inactive` is now deprecated, and it will be removed in a future release.
 
-Introduce a CSS class that can be used to hide any given element from the print view: `govuk-!-display-none-print`
+[Pull request #1711: Additional Tag modifier classes for different colours](https://github.com/alphagov/govuk-frontend/pull/1711).
+
+#### Hide elements when users print a page
+
+You can now hide elements when users print a page, using the new `govuk-!-display-none-print` class.
 
 [Pull request #1723: Add display override for hiding content when printing](https://github.com/alphagov/govuk-frontend/pull/1723).
 
+#### The `iff` Sass function is now deprecated
```

Comment:
Very much non-blocking, thinking out loud… should we consider splitting deprecations out into their own section? This isn't really a new feature.

Reply:
Might be an idea for a future version? I'm not sure about giving them a whole section to themselves, but maybe a subsection somewhere...